examples-new/native-authentication-exchange/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "SECRET_KEY": "replace me with a random string",
+  "NYLAS_OAUTH_API_ID": "replace me with the API ID from Nylas",
```

Comment:
I named them that way for consistency with the other two examples. The "hosted-oauth" example uses those names to set up OAuth via Flask-Dance: https://flask-dance.readthedocs.io/en/latest/providers.html#module-flask_dance.contrib.nylas

Do you want me to make this example different from the others? Do you want me to use a different name in all of them? That will require pushing out a new version of Flask-Dance with that update, as well.